src/system.jl

```diff
@@ -99,6 +99,94 @@ struct Branch
     break_points::Tuple{Float64, Float64}
     "Price penalties for each of the break points of the branch (\$)"
     penalties::Tuple{Float64, Float64}
+    "Resistance of the transformer (pu)"
+    resistance::Float64
+    "Reactance of the transformer or branch (pu)"
+    reactance::Float64
```

Comment:
Given that transformer is a subset of branch, I'd just say "branch" here – "transformer or branch" is confusing

I'd also say branch for the resistance as it makes sense for lines to have resistance even though we don't use it in the formulation or to compute shift factors